os/include/tinyara/log_dump/log_dump.h

```diff
@@ -41,8 +41,9 @@
  * Public Functions
  ********************************************************************************/
 
-#define OPEN_LOGDUMP()			open("/proc/logsave", O_RDWR)
-#define START_LOGDUMP_SAVE(fd)		write(fd, LOGDUMP_SAVE_START, strlen(LOGDUMP_SAVE_START) + 1)
-#define STOP_LOGDUMP_SAVE(fd)		write(fd, LOGDUMP_SAVE_STOP, strlen(LOGDUMP_SAVE_STOP) + 1)
-#define READ_LOGDUMP(fd, buf, bufsize)	read(fd, buf, bufsize)
-#define CLOSE_LOGDUMP(fd)		close(fd)
+#define OPEN_LOGDUMP()				open("/proc/logsave", O_RDWR)
+#define START_LOGDUMP_SAVE(fd)			write(fd, LOGDUMP_SAVE_START, strlen(LOGDUMP_SAVE_START) + 1)
+#define STOP_LOGDUMP_SAVE(fd)			write(fd, LOGDUMP_SAVE_STOP, strlen(LOGDUMP_SAVE_STOP) + 1)
+#define SET_LOGDUMP_FETCH_BLOCKS(fd, num)	write(fd, num, strlen(num) + 1)
```

Comment:
How about changing them to treat an integer value instead of a string?
I think you made this because you tested it with TASH. But app should call this functions so that using integer is better than using string. 
The log dump example should convert string to int and then should call log dump macros.

Reply:
This could be done with new pr.

Reply:
oh. we already use this with string. Please ignore my comment.

Reply:
I am confused. If an app uses the macros above, then we can change it with integer instead of string. It will be better for performance and for code. Could you check it?

Reply:
Done, used integer instead.